pkg/trace/agent/agent.go

```diff
@@ -303,9 +303,11 @@ func (a *Agent) ProcessStats(in pb.ClientStatsPayload, lang, tracerVersion strin
 		}
 	}
 	out := pb.StatsPayload{
-		Stats:         []pb.ClientStatsPayload{in},
-		AgentEnv:      a.conf.DefaultEnv,
-		AgentHostname: a.conf.Hostname,
+		TracerAggregated: true,
```

Comment:
Since we're using the term "client computed stats" throughout. Should this better be `ClientComputed` instead of `TracerAggregated` ?

Reply:
Good idea.